pow/randomx/sys/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "kulupu-randomx-sys"
 description = "FFI bindings for RandomX"
-license = "BSD-3-Clause"
+license = "GPL-3.0-or-later"
```

Comment:
Those packages do not have any other contributors other than me. I changed the license here from `BSD-3-Clause` to `GPL-3.0-or-later` to keep the whole repository unified in regards of licensing.